.github/workflows/release.yml

```
@@ -19,29 +19,20 @@ jobs:
       - name: Install dependencies
         run: |
-          yarn install
+          pnpm install
 
-      - name: Build
-        run: |
-          yarn clean
-          yarn build
-          cp logo.svg deno/lib
-          cp README.md deno/lib
+     
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)